packages/block-library/src/button/edit.js

```diff
@@ -400,6 +479,16 @@ function ButtonEdit( props ) {
 							} }
 							forceIsEditingLink={ isEditingURL }
 							settings={ LINK_SETTINGS }
+							createSuggestion={
+								createPageEntity && handleCreate
+							}
+							withCreateSuggestion={ userCanCreatePages }
+							createSuggestionButtonText={ createButtonText }
+							showInitialSuggestions
+							suggestionsQuery={ getSuggestionsQuery(
+								'page',
+								'post-type'
+							) }
```

Comment:
```suggestion
```

Let's disable initial suggestions for now and only display them when the user starts typing/searching.